app/models/host_aspect_base.rb

```diff
@@ -0,0 +1,25 @@
+class HostAspectBase < ActiveRecord::Base
```

Comment:
There is no relation corresponding to `HostAspect.execution_model` here, is that intentionally?